gallery/src/styles/NxReadOnly/NxReadOnlyPage.tsx

```diff
@@ -58,14 +76,37 @@ const NxReadOnlyPage = () => {
                 displayed can be any string or HTML markup.
               </td>
             </tr>
+            <tr className="nx-table-row">
+              <td className="nx-cell"><code className="nx-code">.nx-read-only__item</code></td>
+              <td className="nx-cell"><code className="nx-code">NxReadOnly.Item</code></td>
+              <td className="nx-cell">
+                Child of <NxCode>.nx-read-only--grid</NxCode>, parent of <NxCode>.nx-read-only__label</NxCode> and
+                {' '}<NxCode>.nx-read-only__data</NxCode>.
+              </td>
+              <td className="nx-cell">
+                When creating an <NxCode>.nx-read-only</NxCode> with a grid layout,
+                each <NxCode>&lt;dt&gt;</NxCode>/<NxCode>&lt;dd&gt;</NxCode> group must be wrapped
+                in a div with this class. This class/element should not be used in non-grid layouts.
+              </td>
+            </tr>
           </tbody>
         </table>
       </GalleryDescriptionTile>
-      <GalleryExampleTile title="Styling read-only lists of data"
+      <GalleryExampleTile title="Simple Example"
+                          id="nx-read-only-simple-example"
                           codeExamples={nxReadOnlyDlExampleCode}
                           htmlExample={nxReadOnlyDlExampleCode}>
         Standard <code className="nx-code">nx-read-only</code> layouts, by default all text wraps.
       </GalleryExampleTile>
+
+      <GalleryExampleTile title="Grid Example"
+                          id="nx-read-only-grid-example"
+                          codeExamples={gridCode}
+                          htmlExample={nxReadOnlyGridExampleCode}>
+        An <NxCode>nx-read-only</NxCode> arranged in a grid. The exact layout of the grid is custom to this example.
+        In typical usage, the layout of the grid would be usage-specific requiring the use
+        of <NxCode>grid-template</NxCode> and/or <NxCode>grid-area</NxCode> CSS properties.
+      </GalleryExampleTile>
```

Comment:
I think that it's worth calling out the use of `nx-read-only__item` here as well. 

e.g. "Note the use of `nx-read-only__item` wrapping each set of `dt/dd`s."

Reply:
> I think that it's worth calling out the use of nx-read-only__item here as well.

97c9a682